src/js/contentUtils.ts

```diff
@@ -506,15 +511,18 @@ export const processFoundJS = async (version: string): Promise<void> => {
           pendingScriptCount--;
           const inlineScriptMap = new Map();
           if (response.valid) {
-            inlineScriptMap.set(response.hash!, script.rawjs);
+            inlineScriptMap.set(response.hash, script.rawjs);
```

Comment:
I guess this isn't functionally any different, but I guess it's worth noting that there's a possibility that we're setting the `undefined` key in the map 🤔

I wonder if it's worth checking for a missing hash and short-circuiting / throwing so that we don't end up with some strange maps or logs.

Reply:
Can revisit 👍